data/data/manifests/tectonic/kubeadmin-password-secret.yaml.template

```diff
@@ -0,0 +1,7 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  namespace: kube-system
+  name: kubeadmin
+data:
+  kubeadmin: {{.Base64encodeKubeadminPwHash}}
```

Comment:
nit: `Base64EncodedKubeadminPwdHash`

Reply:
unfortunately there are others: https://github.com/openshift/installer/pull/771/files#diff-8f8c3c8fa8f2086bba77369270017bb7L20  and a few more (I added a few in a previous PR).  I'll modify this here, and I'll go back and modify the others in a follow-up PR.